.azure-pipelines/functionality-test.yml

```diff
@@ -19,13 +19,6 @@ variables:
   value: $(Build.BuildNumber)
```

Comment:
Shall we rename to `function-tests.yml`?

Reply:
how about sdk-core.yml?

Reply:
Hmm.. can't tell it's for tests from the its name.